functions-and-scope.js

```diff
@@ -78,6 +141,24 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Zorg ervoor dat jouw functie ook werkt als we een andere array willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
 
+/*-------------------CODE------------------*/
+const grades = [8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
+
+function highestgrade(cijfers) {
+    let bestgrade = cijfers[0];
```

Comment:
Waarom begin je hier met de initiële waarde van de eerste entry in de array? (in dit geval is dat dus 8).